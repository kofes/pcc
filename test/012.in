function foo(it1, it2 : integer);
procedure bar(it1, it2 : byte);
