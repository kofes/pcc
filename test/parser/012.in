var
ident1, ident2:type
