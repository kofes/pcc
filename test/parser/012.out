Illegal expression: 'DIV' in pos (1, 1);
